src/com/dynatrace/loadrunner/Main.java

```diff
@@ -1,59 +1,202 @@
+		
+		System.out.println("The LoadRunner Request Tagging tool uses the following syntax: \n"
+				+ "java -jar Dt-LoadRunner-request-tagging.jar <mode> <path parameter> <optional parameters> \n"
+				+ "\n"
+				+ "mode:\n"
+				+ "	insert: to add the Dynatrace HTTP header to the selected LoadRunner scripts, type insert\n"
+				+ "	delete: to remove all modifications previously made by the LoadRunner Request Tagging tool, type delete\n"
+				+ "path parameter:\n"
+				+ "	Pick either -path or -body and -header\n"
+				+ "	-path <filepath>: use to scan all directories and subdirectories for script file's and insert/delete script in them\n"
+				+ "	-body <files> -header <files>: use to select exactly in which header and body files should be processed, the file seperator is & between files\n"
+				+ "optional parameteer:\n"
+				+ "	-LSN <value>: sets load script name to value passed after -LSN. If skipped, the script name will be taken from *.usr file\n"
+				+ "	-c: sets C as scripting language used (default)\n"
+				+ "	-js: sets JavaScript as scripting language\n"
+				+ "	-help: prints usage\n");
+	}
+
+	public static void main(String[] args) {
+
+		System.out.println(VERSION + "\n");
+
+		if (args.length == 0) {
+			showUsages();
+			return;
+		}
+
+		CommandLineParser parser = new CommandLineParser(args);
+
+		if (parser.arguments.containsKey("-help") || parser.arguments.containsKey("help")) {
+			showUsages();
+			return;
+		}
+
+		for (Entry<String, String> pair : parser.arguments.entrySet()) {
+			String key = pair.getKey().toLowerCase();
+			if (key.equals("insert")) {
+				getMode(pair.getKey());
+			} else if (key.equals("delete")) {
+				getMode(pair.getKey());
+			} else if (key.equals("lsn")) {
+				getScriptName(pair.getValue());
+			} else if (key.equals("path")) {
+				try {
+					File directory = new File(pair.getValue());
+					searchPath(directory);
+				} catch (NullPointerException e) {
+					System.out.println("Make sure you'd provided correct path to script file \n");
+					return;
+				}
+			} else if (key.equals("-c")) {
+				cEngine = true;
+			} else if (key.equals("-js")) {
+				cEngine = false;
+			} else if (key.equals("body")) {
+				String bodies[] = pair.getValue().split("&");
+				for (String str : bodies) {
+					body.add(new ScriptFile(new File(str)));
+				}
+			} else if (key.equals("header")) {
+				String header[] = pair.getValue().split("&");
+				for (String str : header) {
+					headers.add(new ScriptFile(new File(str)));
+				}
+			} else {
+				showUsages();
+				System.out.println("\nUnknown parameter: " + key);
+				return;
+			}
+		}
+		if (scriptName.isEmpty() && parser.containsKey("path")) {
+			getScriptNameFromPath(parser.arguments.get("path"));
+		}
+		checkEngine();
+		boolean validated = validateParams();
+		if (validated) {
+			LoadRunnerConverter converter = new LoadRunnerConverter();
+			converter.configureConverter(mode, headers, body, scriptName, cEngine);
+			converter.convertFiles();
```

Comment:
there should be two `convert()` methods:
1. `convert(String path)`
2. `convert(Collection<ScriptFile> headerFiles, Collection<ScriptFile> bodyFiles)`

reflecting two different usage scenarios